assets/stylesheets/common/chat-retention-reminder.scss

```diff
@@ -1,7 +1,7 @@
 .chat-retention-reminder {
   display: flex;
   position: absolute;
-  top: 15px;
+  top: 0;
```

Comment:
This is because the join button could be there and the interaction is better with `top: 0` if the join button is present as well. The banner only shows up in very edge cases, so I wanted to optimize for the join button which is much more likely. It's not trivial to move one based on the other's visibility also